### DISCUSSION TITLE
UE 5.4

### DISCUSSION THREAD

**Discussion by KazeeKEN** [metaworking/channeld-ue-plugin#102] [General]
Hello! When update for Unreal Engine 5.4?

**Comment by HeadClot** [comment]
Just want to echo this. I am on Unreal 5.4.4

**Reply by indiest** [reply]
Short answer: shortly after 5.5 is officially released.

Long answer: Due to short of hands, I can only start working on the update when a minor version of the engine is stable. Otherwise I may need spend more time if a patch release breaks the plugin.

Sorry that it has to go this way. I could use some help here, like, letting me know what does not work for the newer version of the engine. Submming a PR is even better!

**Comment by indiest** [comment]
Hi @KazeeKEN @HeadClot , the branch that supports UE 5.4 is [here](https://github.com/channeldorg/channeld-ue-plugin/tree/ue-5.4). You are welcome to check out and test!

**Reply by HeadClot** [reply]
Thank you for the tag @indiest :)
